### PR TITLE
Updated the interface of scanner(#1wfbq8e)

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

Screenshot from 2021-12-16 17-48-54

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [X] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by disha1202** [comment]
Screenshot from 2021-12-17 13-37-33

**Comment by disha1202** [comment]
Screenshot from 2021-12-17 13-40-06

**Comment by disha1202** [comment]
Screenshot from 2021-12-17 13-41-00

**Comment by disha1202** [comment]
Screenshot from 2021-12-17 17-58-44
Screenshot from 2021-12-17 17-59-09

File: src/components/Scanner.vue [hunk 1]
```diff
@@ -6,41 +6,88 @@
...
+    return{
+      flag:false,
+    }
+  },
   methods: {
-    onDecode (result) {
-      modalController.dismiss({dismissed: true, value: result});
-      result = ''
+   onDecode (result) {
+     if(this.flag === true){
```
Comment (ymaheshwari1) [inline src/components/Scanner.vue:48]:
```suggestion
     if(this.flag){
```

File: src/components/Scanner.vue [hunk 2]
```diff
@@ -6,41 +6,88 @@
...
+       console.log(result);
```
Comment (ymaheshwari1) [inline src/components/Scanner.vue:49]: Remove all console log statements

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]
Improve indentation

File: src/components/Scanner.vue (hunk 1 above)
Comment (disha1202) [reply src/components/Scanner.vue]: Improved indentation

File: src/components/Scanner.vue (hunk 2 above)
Comment (disha1202) [reply src/components/Scanner.vue]: Done sir

File: src/components/Scanner.vue (hunk 1 above)
Comment (ymaheshwari1) [reply src/components/Scanner.vue]: @disha1202 I think this is not done

File: src/components/Scanner.vue [hunk 3]
```diff
@@ -6,41 +6,86 @@
...
+        <ion-icon :icon="scanOutline" />
+      </ion-fab-button>
+    </ion-fab>
+  </ion-content>
 </template>
 
 <script>
+import emitter from "@/event-bus"
 import { StreamBarcodeReader } from "vue-barcode-reader";
 import { IonButton, IonButtons, IonIcon, IonToolbar, modalController } from '@ionic/vue';
```
Comment (ymaheshwari1) [inline src/components/Scanner.vue:26]: Please import all the components used on the page, here `ion-fab`, `ion-content`, and `ion-fab-button` are missing.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/components/Scanner.vue (hunk 3 above)
Comment (disha1202) [reply src/components/Scanner.vue]: Done.

**Review by ymaheshwari1 — APPROVED** [review]
LGTM

**Comment by adityasharma7** [comment]
Closing as will be done differently

**Pull request closed without merging**